lib/isDecimal.js

```diff
@@ -13,8 +13,9 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 
 var decimal = /^[-+]?([0-9]+|\.[0-9]+|[0-9]+\.[0-9]+)$/;
 
-function isDecimal(str) {
+function isDecimal(str, options) {
   (0, _assertString2.default)(str);
-  return str !== '' && decimal.test(str);
+  options = options || {};
+  return str !== '' && decimal.test(str) && (!options.hasOwnProperty('digitsAfterDecPoint') || str.indexOf('.') === -1 || str.length - str.indexOf('.') - 1 <= options.digitsAfterDecPoint);
```

Comment:
- Can fix this line length to be `<80 cols`.
- I don't know if `fractionalDigits` will be a better name than `digitsAfterDecPoint`? [See this](https://math.stackexchange.com/questions/64042/what-are-the-numbers-before-and-after-the-decimal-point-referred-to-in-mathemati)